### DISCUSSION TITLE
Traffic monitoring on Home Lab

### DISCUSSION THREAD

**Discussion by 4th3LuLz0fIt** [Security-Onion-Solutions/securityonion#5228] [Unsupported Versions]
I set up an SO in VMware workstation and would like to know how to monitor traffic on my local network?

**Comment by innovate-support** [comment]
You have to have an interface that the SO VM can see the traffic that's flowing on your network.  In VM speak that's called 'promiscuous mode'.  In a physical switch it would be called a 'mirror port', otherwise known as a 'tap'.  Research those concepts and it will help you figure out how to feed your traffic into your VM.

**Reply by 4th3LuLz0fIt** [reply]
I think I need the "tap", promiscuous mode runs automatically as vmnet0 once you run vmware as an admin in win10.

Thanks, I'll look into setting up tap.
